benchmark/worker/benchmark_client.go

```diff
@@ -309,7 +304,7 @@ func (bc *benchmarkClient) streamingLoop(conns []*grpc.ClientConn, rpcCountPerCo
 		// For each connection, create rpcCountPerConn goroutines to do rpc.
 		for j := 0; j < rpcCountPerConn; j++ {
 			c := testgrpc.NewBenchmarkServiceClient(conn)
-			stream, err := c.StreamingCall(context.Background())
+			stream, err := c.StreamingCall(ctx)
```

Comment:
We should revert to using context.Background for the stream. See https://github.com/grpc/grpc-go/pull/8614#issuecomment-3414787984